Lusas_Adapter/Convert/ToBHoM/Loads/ToGravityLoad.cs

```diff
@@ -48,11 +48,19 @@ public static partial class Convert
             Loadcase loadcase = ToLoadcase(assignedLoadcase);
             Vector gravityVector = new Vector
             {
-                X = lusasGravityLoad.getValue("accX"),
-                Y = lusasGravityLoad.getValue("accY"),
-                Z = lusasGravityLoad.getValue("accZ")
+                X = lusasGravityLoad.getValue("accX") / 9.80665,
+                Y = lusasGravityLoad.getValue("accY") / 9.80665,
+                Z = lusasGravityLoad.getValue("accZ") / 9.80665
```

Comment:
As above, consider having `g` as a global variable as it will be used in the push and pull, and the user may want to set it themselves.